numba_cuda/numba/cuda/tests/cudapy/test_ir_utils.py

```diff
@@ -47,11 +47,15 @@ def mk_pipeline(
                     locals = {}
                 if not flags:
                     flags = Flags()
-                flags.nrt = True
+                # flags.nrt = True
```

Comment:
what happens if this is enabled?

Reply:
Okay, I confirmed with Atmn and Vijay. Graham left this comment in our spreadsheet with respect to NRT:

> GM: We should not be vendoring this, references to it in the code should be removed

So I think it's safe to remove this commented-out line.

To answer your specific question _what happens if this is enabled?_, we get a runtime error because `flags.nrt` is immutably-inherited from Numba's configuration.

How does this sound to you?